change locale names

fryderykg opened the pull request:
chnage names to en-gb and pl_pl and 
related uk names to gb, 
change flag names to gb.gif and pl.gif

fryderykg committed:
change locale name to en-gb and pl_pl and related uk names to gb, change flag names to gb.gif and pl.gif

Johny1010 commented on index.html:
```diff
@@ -11,23 +11,23 @@
+          <a href="#" class="flag_gb" id="flag_gb"><img src="pics/gb.GIF" alt="great britain flag"></a>
         </div>
       </header>
       <div class="opacityZero hidden" id="board">
```
Code standards (all classes should be written with "-" between words) 
e.g class="opacity-zero"

fryderykg replied:
Ok zmieniłem nazwy klas oraz nazwy locale oraz dodałem grunta do projektu

fryderykg committed:
change flag extension from uppercase .GIF to lowercase .gif and appropriate path in index.html

fryderykg committed:
add grunt to project and initial files

fryderykg committed:
add sass, imagemin and watch configuration to grunt

fryderykg committed:
rename style.sass to main.sass
minify picture with grunt and put it into build subfolder
compile sass with grunt

fryderykg committed:
add minified pictures to repository

fryderykg committed:
change camelCase class names to names with hyphen

fryderykg commented:
pozmieniałem nazwy klas i dostosowałem projekt do grunta
